amethyst_input/src/input_handler.rs

```diff
@@ -17,7 +17,7 @@ use super::event::InputEvent::*;
 /// For example, if a key is pressed on the keyboard, this struct will record
 /// that the key is pressed until it is released again.
 #[derive(Derivative)]
-#[derivative(Default(bound = "AX: Hash + Eq, AC: Hash + Eq"))]
+#[derivative(Default(bound = ""))]
```

Comment:
why did you keep bound?

Reply:
Because otherwise it would require the full bounds of the actual type.